src/main/java/com/josdem/vetlog/service/impl/PetServiceImpl.java

```diff
@@ -64,14 +64,13 @@ public Pet update(Command command) throws IOException {
         recoveryImages(petCommand);
         Pet pet = petBinder.bindPet(petCommand);
         Optional<User> user = getUser(petCommand.getUser());
-        if (!user.isPresent()) {
+        user.ifPresentOrElse(value -> pet.setUser(value), () -> {
             throw new BusinessException(NO_USER_WAS_FOUND_WITH_ID + petCommand.getUser());
```

Comment:
I suggest using method class reference, since it is cleaner:
`user.ifPresentOrElse(pet::setUser, () -> {`